tests/integration/functions.py

```diff
@@ -573,7 +573,7 @@ def fib_to_7_seg(n):
 
 def callee(a: int, b: int) -> int:
     c = a + b
-    return c
+    return 5
```

Comment:
is this a typo

Reply:
No, I have just modified that test to return a literal to check that what this PR does is correct.